src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -250,6 +268,26 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
         return this.state.zxcvbnResult && this.state.zxcvbnResult.score >= PASSWORD_MIN_SCORE;
     }
 
+    _renderPhaseMigrate() {
+        // This is a temporary screen so people who have the labs flag turned on and
```

Comment:
Can we stick an XXX or TODO on this? (and/or track it in an issue to remind ourselves to clean up)

Reply:
Good idea